### PR TITLE
[T4A2][F09-B2]AntoniaDevina

### PR DESCRIPTION
**Pull request by antoniadevina**

### CODE REVIEW

**Comment by antoniadevina** [comment]
ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -25,6 +28,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = nextSequenceNumber;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java:31]: it should start from 1, not 0.

File: src/seedu/addressbook/data/person/Person.java [hunk 2]
```diff
@@ -10,11 +10,14 @@
  */
 public class Person implements ReadOnlyPerson {
 
+	private static int nextSequenceNumber = 0;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java:13]: Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments. Pay attention to the commit before submitting your PR, since commit in this PR is mixed with commit for other task.

**Pull request closed without merging**